travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls pylint Click docutils git+https://github.com/moylop260/pylint_oca.git > /dev/null 2>&1
```

Comment:
It's noce to have pylint_oca should as a separate tools; IMO it should be moved under OCA.